docker/dev/Dockerfile

```diff
@@ -58,6 +58,13 @@ RUN apt-get update && \
 RUN groupadd -g 1000 ubuntu && \
     useradd -u 1000 -g ubuntu -m -N -s /bin/bash ubuntu
 
+# We are adding sudo capabilities to the ubuntu user so that devs can run the cmake build inside the container.
```

Comment:
We need to add sudo capabilities to allow the dev to rebuild within the container